packages/core/src/types-hoist/log.ts

```diff
@@ -1,6 +1,6 @@
 import type { ParameterizedString } from './parameterize';
 
-export type LogSeverityLevel = 'trace' | 'debug' | 'info' | 'warn' | 'error' | 'fatal' | 'critical';
+export type LogSeverityLevel = 'trace' | 'debug' | 'info' | 'warn' | 'error' | 'fatal';
```

Comment:
this is technically breaking 🤔 but if it breaks, I guess it is necessary?

Reply:
Nothing should be using this yet, and yup we need to change it.